### ISSUE TITLE
Error using  pkgreviewr::pkgreview_create() - review.md not found

### ISSUE THREAD

**Issue by beatrizmilz** [ropensci-org/pkgreviewr#72]
Hi! I was trying to create the template for reviewing using `pkgreviewr::pkgreview_create()`, and got an error. The version of the package that is installed is the most recent from GitHub.

The error was: **URL not found: https://api.github.com/repos/ropensci/software-review/contents/.buffy/templates/review.md**
Seems like the function is trying to access the file `.buffy/templates/review.md` from this repo: https://github.com/ropensci/software-review/tree/master/.buffy/templates . But there is no such file there. 

Looking at the commit history, seems like the file `review.md` was deleted 12 days ago: https://github.com/ropensci/software-review/commit/228346b68515fe4e73deb76d60d7ca0d3844289f#diff-d680c2407256f0bfc1312efc544ac2fa3a7f97d2fbb242a89043b8b664b1c54b

This file is meant to stay deleted?

Full message: 
```
> pkgreviewr::pkgreview_create(pkg_repo = "frictionlessdata/frictionless-r", 
+                              review_parent = ".")
cloning into '/var/folders/z5/nh0wlcsj2fq99rjrj1x7hcwc0000gn/T/Rtmp6Mz0ma/pkgreviewr_prep/frictionless-r'...
Receiving objects:   1% (42/4135),   48 kb
Receiving objects:  11% (455/4135),  472 kb
Receiving objects:  21% (869/4135),  536 kb
Receiving objects:  31% (1282/4135),  601 kb
Receiving objects:  41% (1696/4135),  649 kb
Receiving objects:  51% (2109/4135),  689 kb
Receiving objects:  61% (2523/4135),  729 kb
Receiving objects:  71% (2936/4135),  776 kb
Receiving objects:  81% (3350/4135),  816 kb
Receiving objects:  91% (3763/4135),  888 kb
Receiving objects: 100% (4135/4135), 1072 kb, done.
✔ Package source cloned successfully
✔ pkg_dir written out successfully
Error in gh_process_response(raw) : 
GitHub API error (404): 
Message: Not Found
Read more at https://docs.github.com/rest/reference/repos#get-repository-content

URL not found: https://api.github.com/repos/ropensci/software-review/contents/.buffy/templates/review.md
Type .Last.error.trace to see where the error occurred
```

**Comment by annakrystalli** [comment]
Hello @beatrizmilz ! Many thanks for taking the time to report this issue! Will investigate and correct.

**Comment by annakrystalli** [comment]
This has now been fixed in version 0.3.0

**Comment by beatrizmilz** [comment]
Thank you Anna! I updated the package and tried here, and it works! Thanks :tada:
